package.json

```diff
@@ -8,9 +8,10 @@
     "@wireapp/react-ui-kit": "7.30.3",
     "@wireapp/store-engine-dexie": "1.3.9",
     "@wireapp/store-engine-sqleet": "1.3.12",
-    "@wireapp/webapp-events": "0.3.0",
+    "@wireapp/webapp-events": "0.4.0",
     "amplify": "git+https://git@github.com/wireapp/amplify",
     "core-js": "2.6.10",
+    "countly-sdk-nodejs": "20.4.0",
```

Comment:
Probably we'll have to use this: https://support.count.ly/hc/en-us/articles/360037441932-Web-analytics-JavaScript-